.github/workflows/pull-request-checks.yaml

```diff
@@ -378,3 +378,28 @@ jobs:
           docker run -v ${PWD}/.github/workflows/smoke_test_assets:/mnt/smoke -t cbmc jbmc --classpath /mnt/smoke Test
       - name: Smoke test goto-analyzer
         run: docker run -v ${PWD}/.github/workflows/smoke_test_assets:/mnt/smoke -t cbmc goto-analyzer /mnt/smoke/test.goto --unreachable-functions
+
+  codecov-coverage-report:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Download testing and coverage dependencies
+        run: sudo apt-get install --no-install-recommends -y g++ flex bison cmake maven jq libxml2-utils dpkg-dev openjdk-11-jdk-headless lcov
```

Comment:
I don't think we need `dpkg-dev` for this build?

Reply:
You are right on this as well.